.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -9,6 +9,19 @@ body:
       options:
         - label: "My problem is not solved in the Troubleshooting docs"
           required: true
+  - type: dropdown
+    id: mamba_installation_method
+    attributes:
+      label: How did you install Mamba?
+      description: |
+        Note that Mamba installation methods other than [Mambaforge](https://mamba.readthedocs.io/en/latest/installation.html#fresh-install) are **NOT SUPPORTED**.
+
+        If this issue is about Micromamba, please select "Other".
```

Comment:
Perhaps, I would rather say something like, *if you use mamba* you need to install through mambaforge.